src/store/Account.js

```diff
@@ -102,6 +102,11 @@ const getters = {
   ethWalletAddress (state) {
     let wallet = find('ethereum_wallet', state.accountInfo)
     return wallet ? wallet.ethereum_wallet : 'no eth deposit address'
+  },
+
+  withdrawWalletAddresses (state) {
+    const wallet = find('eth_whitelist', state.accountInfo)
+    return wallet ? [wallet.eth_whitelist] : []
```

Comment:
It seems that only one address can be set, but I suspect that multiple addresses must be required. As far as I see, Notary verifies whether an address is whitelisted at [this line](https://github.com/d3ledger/notary/blob/f32e9887ca3cd5da3556c9110d29cd765d21680a/src/main/kotlin/notary/endpoint/eth/EthRefundStrategyImpl.kt#L143) with splitting eth_whitelist with `,`. So maybe we should split it as well?